test.js

```diff
@@ -0,0 +1,12 @@
+const fauna = require('./index')
+const { query: q } = fauna
+const db = new fauna.Client({
+  secret: 'fnADrL7VfSACAhtIplPXl_C-iX20yk6hQbReCxAI',
+})
+
+;(async () => {
+  // let res = await db.query(q.Let({ super: true }, q.Var('super')))
```

Comment:
@gahabeen If they're not needed, can you please remove these comments? Thanks!

Reply:
Damned, these have slipped in the push. My bad 😅
I'll remove them.

Reply:
Done 👌

Reply:
Thank you! 🙏